notebooks/labs37_notebooks/SentimentAnalysisFor_TwitterBot.ipynb

```diff
@@ -666,24 +2516,265 @@
+            "application/vnd.jupyter.widget-view+json": {
+              "model_id": "08c1bdcbdac54cb280764e496f26c47d",
+              "version_minor": 0,
+              "version_major": 2
+            },
+            "text/plain": [
+              "Downloading:   0%|          | 0.00/268M [00:00<?, ?B/s]"
+            ]
+          },
+          "metadata": {}
+        },
+        {
+          "output_type": "display_data",
+          "data": {
+            "application/vnd.jupyter.widget-view+json": {
+              "model_id": "3bc7f3d39b9041c983afdc81d3ed00e0",
+              "version_minor": 0,
+              "version_major": 2
+            },
+            "text/plain": [
+              "Downloading:   0%|          | 0.00/48.0 [00:00<?, ?B/s]"
+            ]
+          },
+          "metadata": {}
+        },
+        {
+          "output_type": "display_data",
+          "data": {
+            "application/vnd.jupyter.widget-view+json": {
+              "model_id": "92ffcb8f301d43eb9157c1f2c412d247",
+              "version_minor": 0,
+              "version_major": 2
+            },
+            "text/plain": [
+              "Downloading:   0%|          | 0.00/232k [00:00<?, ?B/s]"
+            ]
+          },
+          "metadata": {}
+        }
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "3QfXVuzwuzT1"
+      },
+      "source": [
+        "# Test data\n",
+        "pos_text = \"Yeah would love to do this for you\"\n",
+        "neg_text = \"Hell no, I'm not talking to no damn bot\""
+      ],
+      "execution_count": 3,
       "outputs": []
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "id": "OKeVYddGu1bX",
+        "outputId": "5336ac51-dce0-4e8e-e1bf-ce8cdd86788c"
+      },
+      "source": [
+        "# Test Results\n",
+        "\n",
+        "result = sentiment_analysis(pos_text)[0]\n",
+        "print(\"Label:\", result['label'])\n",
+        "print(\"Confidence Score:\", result['score'])\n",
+        "print()\n",
+        "\n",
+        "result = sentiment_analysis(neg_text)[0]\n",
+        "print(\"Label:\", result['label'])\n",
+        "print(\"Confidence Score:\", result['score'])"
+      ],
+      "execution_count": 4,
+      "outputs": [
+        {
+          "output_type": "stream",
+          "text": [
+            "Label: POSITIVE\n",
+            "Confidence Score: 0.9996284246444702\n",
+            "\n",
+            "Label: NEGATIVE\n",
+            "Confidence Score: 0.9990489482879639\n"
+          ],
+          "name": "stdout"
+        }
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "40dljUClvHYQ"
+      },
+      "source": [
+        "Even though the confidence score is superb, it's important to note that the responses that have \"F yeah\" are categorized negative which is incorrect"
+      ]
     }
   ]
 }
```

Comment:
Again, just being picky here but please leave an empty line after the final line in code.